hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/cli/GenericCli.java

```diff
@@ -85,12 +89,17 @@ public int execute(String[] argv) {
   }
 
   protected void printError(Throwable error) {
-    //message could be null in case of NPE. This is unexpected so we can
-    //print out the stack trace.
-    if (verbose || Strings.isNullOrEmpty(error.getMessage())) {
-      error.printStackTrace(cmd.getErr());
+    if (error instanceof FileSystemException) {
+      String errorMessage = handleFileSystemException((FileSystemException) error);
+      cmd.getErr().println(errorMessage);
```

Comment:
Let's keep the original `if-else`, and move handling of `FileSystemException` to the `else` block.  In other words, `verbose` mode should still print the stack trace for `FileSystemException`.